verification/curator-service/ui/src/components/ViewCase.test.tsx

```diff
@@ -23,17 +24,24 @@ it('loads and displays case', async () => {
     mockedAxios.get.mockResolvedValueOnce(axiosResponse);
 
     const { findByText, getByText } = render(
-        <ViewCase
-            id="abc123"
-            onModalClose={(): void => {
-                return;
-            }}
-        />,
+        <MemoryRouter>
+            <ViewCase
+                id="abc123"
+                enableEdit={true}
+                onModalClose={(): void => {
+                    return;
+                }}
+            />
+        </MemoryRouter>,
     );
     expect(mockedAxios.get).toHaveBeenCalledTimes(1);
     expect(mockedAxios.get).toHaveBeenCalledWith('/api/cases/abc123');
+    // Case is editable.
+    expect(await findByText('Edit')).toBeInTheDocument();
```

Comment:
Could you expect that Edit isn't in the document in another test?

Reply:
done